.github/workflows/e2e-testing.yml

```
@@ -26,9 +26,22 @@ jobs:
       - name: Build
         run: pnpm run build
 
-      - name: Cypress run
-        uses: cypress-io/github-action@v5
+      - name: Cache Cypress binary
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)